sklearn/cluster/hierarchical.py

```diff
@@ -560,12 +563,29 @@ def _hc_cut(n_clusters, children, n_leaves):
     n_leaves : int
         Number of leaves of the tree.
 
+    distance_threshold : int (optional)
+        The distance threshold to cluster at.
+        If ``distance_threshold`` is set then ``n_clusters`` will be the
```

Comment:
It is confusing here to refer to the `n_clusters` parameter. I'd rather: "If ``distance_threshold`` is set then  `n_clusters` will be ignored. Instead, the number of clusters will be determined by when distances between clusters first exceed the `distance_threshold` during agglomeration."